lib/temporal/workflow/count_workflows_aggregation.rb

```diff
@@ -0,0 +1,11 @@
+module Temporal
+  class Workflow
+    class CountWorkflowAggregation
```

Comment:
ooc, why "Aggregation" as the suffix instead of a more neutral "Result" or "Response"? Does aggregation mean it is aggregating all of the fields in the `CountWorkflowExecutionsResponse`, or is it a reference to the `CountWorkflowExecutionsResponse.AggregationGroup` type?